sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -44,6 +44,13 @@
 
 #----- Microsoft SQLServer 2008/2012
 # Collation must be case-sensitive (CS) and accent-sensitive (AS).
+#Use the following connection string if you want to use integrated security with MS Sql Server.
+#Do not set sonar.jdbc.username or sonar.jdbc.password property if you are using Integrated Auth
+#For Integrated Security to work, you have to download the MS SQL JDBC driver package from http://www.microsoft.com/en-us/download/details.aspx?displaylang=en&id=11774
+#and copy sqljdbc_auth.dll to your path. If you have to copy the 32 bit or 64 bit version of the dll depending upon the architecture of your server machine
+#sonar.jdbc.url=jdbc:sqlserver://localhost;databaseName=sonar;integratedSecurity=true
+
+#Use the following connection string if you want to use SQL Auth while connecting to MS Sql Server. Set the sonar.jdbc.username and sonar.jdbc.password appropriately
 #sonar.jdbc.url=jdbc:sqlserver://localhost;databaseName=sonar
 
```

Comment:
Hi Koushik, the credential properties should not have values "sonar" at lines 14-15:
#sonar.jdbc.username=
#sonar.jdbc.password=